framework/ui/internal/uiengine.h

```diff
@@ -38,11 +38,11 @@ class UiEngine : public QObject, public IUiEngine
 {
     Q_OBJECT
 
-    Q_PROPERTY(QmlTheme* theme READ theme NOTIFY themeChanged)
-    Q_PROPERTY(QmlToolTip* tooltip READ tooltip CONSTANT)
+    Q_PROPERTY(QmlTheme * theme READ theme NOTIFY themeChanged)
+    Q_PROPERTY(QmlToolTip * tooltip READ tooltip CONSTANT)
```

Comment:
adding a space here???

Reply:
Unfortunately, the tool does not recognize this expression :(

Reply:
Great :-(